### DISCUSSION TITLE
Invalid copilot token: missing token: 403. GitHub Copilot could not connect to server. Extension activation failed: "Contact Support. You are currently logged in as GITHUB_USERNAME

### DISCUSSION THREAD

**Discussion by binhtruong9418** [community/community#65728] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

i found this error when sign out and login again in vscode. now i cannot use github copilot. i have github student package

![image](https://github.com/community/community/assets/53460337/a781b0bf-8cd9-4bc7-9638-413b814e36fe)

![github-error (2)](https://github.com/community/community/assets/53460337/e57fde41-c452-4fcd-bdc8-3b5bf700636a)

![github-error](https://github.com/community/community/assets/53460337/660da3a7-0cc7-4e39-b3da-3f15678e79fd)

**Comment by ChristLZS** [comment]
me too

**Comment by davecheney** [comment, marked as answer]
We cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
